frontend/src/modules/processMeasurements/actions.js

```diff
@@ -31,6 +32,21 @@ export const list = (options) => async (dispatch, getState) => {
     ));
 };
 
+export const listFilter = ({ offset = 0, limit = DEFAULT_PAGINATION_LIMIT, filters = {}, sortBy }, abort) => async (dispatch, getState) => {
+    if (getState().processMeasurements.isFetching && !abort)
+        return
+
+    limit = getState().pagination.pageSize;
+    filters = serializeFilterParams(filters, PROCESS_MEASUREMENT_FILTERS)
+    const ordering = serializeSortByParams(sortBy)
+    const options = { limit, offset, ordering, ...filters}
+
+    return await dispatch(networkAction(LIST_FILTER,
+        api.processMeasurements.list(options, abort),
+        { meta: { ...options } }
+    ));
+};
+
```

Comment:
Should I look into creating a High-Order Function so we can replicate this for each models?
Not sure how often we will repeat this though.

Reply:
Not now 👎🏼